trunk/source/BA_FeatureUltimateCommon/feature.xml

```diff
@@ -126,10 +126,6 @@
          id="com.github.jhoenicke.javacup"
          version="0.0.0"/>
 
-   <plugin
-         id="org.apache.commons.lang"
-         version="0.0.0"/>
-
```

Comment:
It is correct to remove this here, but it should be added as a general dependency under `BA_FeatureDependenciesCommon/feature.xml`. Otherwise the products will be packaged incorrectly.

Reply:
Okay, that's somethint I don't quite understand: Do all the external dependencies belong to `BA_FeatureDependenciesCommon/feature.xml`? Currently this is not the case, none of the dependencies in the target-file are mentioned there.

And what do you meant with "packaged incorrectly"? I suppose that the SnakeYAML is currently working anyways in the SV-COMP 😉